src/tezos_interop/tezos_js_bridge.js

```diff
@@ -137,7 +137,9 @@ const read = (callback) => {
       return done();
     },
   });
-  const errorHandler = (err = "close") => failure(err);
+  const errorHandler = (err) => {
+    if (err != null) failure(err);
```

Comment:
What is the purpose of this change @ulrikstrid ?

Reply:
This callback should never be called, as js bridge is permanently lived. I think the only case where this could happen is if you're killing the node, in which case I'm not sure what is the right behavior.

And if this callback is called it should definitely kill the bridge, even if not with a failure.

I would like to avoid this, if the only argument pro it is that it supresses some noise

Reply:
It's not forever lived when using the CLI and there is noise that wrecks the json which is the whole point of this PR @EduardoRFS

Reply:
Actually, we already are printing to stdout. So this PR confuses me a bit. I'm going to minimize it to the smallest change that lets us get rid of `sed` in sandbox.sh.